lib/promscrape/client.go

```diff
@@ -49,28 +49,42 @@ func newClient(ctx context.Context, sw *ScrapeWork) (*client, error) {
 	setProxyHeaders := func(_ *http.Request) error {
 		return nil
 	}
+	dialFunc := netutil.NewStatDialFunc("vm_promscrape")
```

Comment:
nit: would be great to const `vm_promscrape` value, because it looks like a word where it is easy to make a typo. And since we repeat it twice - the chances increases.

Reply:
I think, it's fine to keep it as variable. It's located at the same function and shouldn't case much trouble to change it at one place.